Team9Project/src/com/indragie/cmput301as1/ExpenseClaimListActivity.java

```diff
@@ -18,50 +18,33 @@
 package com.indragie.cmput301as1;
 
 
-import java.util.Comparator;
+import java.util.List;
 
+import android.app.ActionBar;
+import android.app.ActionBar.Tab;
 import android.app.AlertDialog;
-import android.app.ListActivity;
 import android.content.DialogInterface;
-import android.content.Intent;
 import android.content.SharedPreferences;
 import android.os.Bundle;
-import android.view.ActionMode;
-import android.view.Menu;
-import android.view.MenuItem;
-import android.view.View;
-import android.widget.AdapterView;
+import android.support.v4.app.FragmentActivity;
 import android.widget.EditText;
-import android.widget.ListView;
 import android.widget.Toast;
 
 /**
- * An activity that presents a list of expense claims.
+ * An activity that initializes tab fragments for displaying claims.
  */
-public class ExpenseClaimListActivity extends ListActivity implements TypedObserver<CollectionMutation<ExpenseClaim>> {
+
+public class ExpenseClaimListActivity extends FragmentActivity implements TypedObserver<CollectionMutation<ExpenseClaim>> {
```

Comment:
This no longer needs to implement `TypedObserver` if all of that code has been implemented inside the fragment